libkbfs/key_manager.go

```diff
@@ -641,8 +641,9 @@ func (km *KeyManagerStandard) Rekey(ctx context.Context, md *RootMetadata, promp
 	if !isWriter {
 		if len(newReaderUsers) > 0 || addNewWriterDevice || incKeyGen {
 			// If we're a reader but we haven't completed all the work, return
-			// RekeyIncompleteError
-			return false, nil, RekeyIncompleteError{}
+			// RekeyIncompleteError.
+			rekeyDone := len(newReaderUsers) > 0
```

Comment:
Well, it means something has changed about the reader itself (a new device, or a revoked device).  It's true that in this case, when Rekey is called twice, the second Rekey won't do anything but rekeyDone will still be true.  Maybe I should pick a better name here.